src/python/WMComponent/WorkflowUpdater/WorkflowUpdaterPoller.py

```diff
@@ -26,6 +31,188 @@
 from WMCore.DAOFactory import DAOFactory
 
 
+def findFiles(path, pat):
```

Comment:
This function is a great candidate to be moved under the `Utils/` python package. Potentially under this module: https://github.com/dmwm/WMCore/blob/master/src/python/Utils/FileTools.py

Reply:
done